scm-webapp/src/main/java/sonia/scm/search/LuceneSearchEngine.java

```diff
@@ -128,7 +129,8 @@ private void exec(Consumer<IndexParams> consumer) {
       indexManager.all()
         .stream()
         .filter(predicate)
-        .map(details -> new IndexParams(details.getName(), resolver.resolve(details.getType())))
+        .filter(details -> resolver.resolve(details.getType()).isPresent())
+        .map(details -> new IndexParams(details.getName(), resolver.resolve(details.getType()).get()))
```

Comment:
If I'm not mistaken this is unnecessary:
```suggestion
        .filter(details -> details.getType() != null)
        .map(details -> new IndexParams(details.getName(), resolver.resolve(details.getType()).get()))
```